plugins/rikki/heroeslounge/components/slothaccount/update.htm

```diff
@@ -238,7 +245,7 @@ <h3>Description</h3>
                     </div>
                     <input class="form-control" placeholder="BattleTag" title="Battle Tag - talk to a moderator on Discord if this has changed" type="text" name="battle_tag" disabled value="{{ __SELF__.sloth.battle_tag }}">
                 </div>
-                <div class="input-group col-lg-2 mb-2 col-md-3">
+                <div class="input-group col-lg-2 mb-3 col-md-4">
```

Comment:
Changing mb-2 to mb-3 here should lead to an additional row with very little content. Was there too little space available?

Reply:
This straightened out the bottom row of updating the user social information, as (almost) every other social "box" had this definition for it's size.

Reply:
Having them being the same size is fine. You should also aim to fill the rows and not have a single dangling small element that gets it own row.

Both goals can be accomplished at the same time here by changing the sizes of other elements

Reply:
I agree, so I'm a bit confused about your statement. I tested both in Firefox and Chrome and do not experience a single dangling element on it's own row. This change only intends to align the boxes on the second row better, which it does in my testing.